cmd/daemon.go

```diff
@@ -51,8 +52,9 @@ func init() {
 func main() {
 	var metricsAddr string
 	var probeAddr string
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
-	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	// We are host networked, we set default to loopback by default
+	flag.StringVar(&probeAddr, "health-probe-bind-address", "127.0.0.1:39300", "The address the probe endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", "127.0.0.1:39301", "The address the metric endpoint binds to.")
```

Comment:
I wanted to be explicit showing that we do not listen on all interfaces but only loopback. We dont want to expose this port outside the node. I figured `:$PORT_NUM` would listen on all interfaces and thats not what I want even as a default for this component because we always have the rbac proxy.

> in case both pods are on the same node right ?

yes and also, we cannot reserve within OCP the same port for two components.